clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -99,3 +110,51 @@ func applyManifests(
 	}
 	return nil
 }
+
+func WaitForPrometheusToBeHealthy(client clientset.Interface) error {
+	klog.Info("Waiting for Prometheus stack to become healthy...")
+	return wait.Poll(
+		checkPrometheusReadyInterval,
+		checkPrometheusReadyTimeout,
+		func() (bool, error) { return isPrometheusReady(client) })
+}
+
+func isPrometheusReady(client clientset.Interface) (bool, error) {
+	raw, err := client.CoreV1().
+		Services("monitoring").
```

Comment:
nit: You added const for namespace name.

Reply:
Thanks, done!